src/Store/Reducers/CVEDetailsPageStore.js

```diff
@@ -29,7 +29,7 @@ export const initialState = Immutable({
 });
 
 // Reducer
-export const CVEReducer = (state = initialState, action) => {
+export const CVEDetailsPageStore = (state = initialState, action) => {
```

Comment:
This store has an action with type `CLEAR_CVES_STORE`. this should be changed to `CLEAR_CVEs_STORE` because of [this](https://github.com/RedHatInsights/vulnerability-ui/pull/437/files#diff-cbaad4675fca76abfbb5602f9784d866R28)

Reply:
the `CVEDetailsPageStore` is one single CVE so it used the action `CLEAR_CVE_STORE` (without s)